portfolio.js

```diff
@@ -13,15 +15,26 @@ Project.prototype.toHtml = function() {
   $newProject.find('a').text(this.title);
   $newProject.attr('.date-created', this.dateCreated);
   $newProject.attr('src', this.image);
-  $newProject.attr('href', this.fileSource); //need to chain .html jQ method on?
+  $newProject.attr('.description', this.description);
+  // $newProject.attr('href', this.fileSource); //need to chain .html jQ method on?
   $newProject.removeClass('template');
   return $newProject;
 };
 
+portfolio.handleMainNav = function() {
+  $('.main-nav').on('click', '.nav-item', function(event) {
+    event.preventDefault();
+    $('.tab-content').hide();
+    $(`#${$(this).data('content')}`).fadeIn(500);
+  });
+};
+
 projectInfo.forEach(function(currentProject) {
   projects.push(new Project(currentProject));
 });
 
 projects.forEach(function(project) {
   $('#projects').append(project.toHtml());
```

Comment:
This will not work because you don't have a toHtml function currently. If you separate your constructor from your DOM manipulation (and put those jQuery methods into a toHtml method), this will work as expected.